CMakeLists.txt

```diff
@@ -23,13 +25,16 @@ include_directories(${SIMPLE_WEB_SERVER_HDR})
 
 
 file(GLOB_RECURSE EXPRESS_SRC "src/*.cpp")
-add_library(express-cpp SHARED ${EXPRESS_SRC})
```

Comment:
Well, I'm ok for static build support as an OPTION, but not as a default, which you instated by doing this. It's ok for the user to say something like -DBUILD_SHARED_LIBS=OFF, but when not specifying any option, it has to be a shared library.